packages/mdc-textfield/foundation.js

```diff
@@ -150,4 +156,20 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       disabled: false,
     };
   }
+
+  registerAutoCompleteHandler_(input) {
+    return new Promise((resolve, reject) => {
```

Comment:
Did you happen to test this in IE11? Unfortunately, I don't think Promises are supported - http://caniuse.com/#feat=promises :( 

If that is the case, you could use a callback-based approach, e.g. `registerAutoCompleteHandler(onRegistered)` where `onRegistered` is your callback function.